### ISSUE TITLE
[Feature Request] Add YouTube Streaming Support

### ISSUE THREAD

**Issue by davidneedham** [crowbartools/Firebot#1497]
**Describe the solution you'd like**
Firebot is far and above the best bot for Twitch streaming. But many Twitch streamers are switching to YouTube. I recently tried streaming on YouTube and Firebots absence was most definitely noticed. I understand that this is a huge undertaking, but I didn't see any other feature requests opened with this request so I figured it's worth the time to formally ask.

**Comment by Firebottle** [comment]
Just to follow up on this one, one major focus of v6 is multi-platform integration. Youtube would probably be one of the major platforms we incorporate as soon as we can.

**Comment by MichaelGame-Dev** [comment]
Is there any update on when this would be available?

**Comment by zunderscore** [comment]
No, we have no updates to share at this time. When we do, we will update this issue.

**Comment by SaintedWhisper** [comment]
With YouTube blowing up lately, I can't wait for V6 to come out!

**Comment by codihuston** [comment]
Is there any documentation anywhere for those who might be interested in contributing in this feature and/or multi-platform support?

**Comment by Zophar1** [comment]
I'm also quite interested in this - just starting dabbling in multi-streaming and it would be neat if there was some sort of YT integration.

**Comment by SaintedWhisper** [comment]
> I'm also quite interested in this - just starting dabbling in multi-streaming and it would be neat if there was some sort of YT integration.

Have they went to V6 yet!
